supabase/migrations/20260816150923_split_mfa_session_and_email_otp_checks.sql

```
@@ -0,0 +1,65 @@
+-- Split MFA session assurance from email OTP first-factor checks.
+-- `aal2` is the source of truth for completed MFA. Email OTP can be an `aal1`
+-- login method, so it must not satisfy the MFA gate used by RLS/admin checks.
+-- Re-applies the fix from 20260608114543 that was lost in the prod baseline squash.
```

SQLFluff (4.2.2)

[error] Line is too long (84 > 80).

(LT05)